lib/messages/builder.js

```diff
@@ -20,7 +20,7 @@ function builder(options) {
   options.Transaction = options.Transaction || dashcore.Transaction;
   options.MerkleBlock = options.MerkleBlock || dashcore.MerkleBlock;
   options.MnListDiff = options.MnListDiff || dashcore.MnListDiff;
-  options.protocolVersion = options.protocolVersion || 130000;
+  options.protocolVersion = options.protocolVersion || 70215;
```

Comment:
Based on naming I think this is correct, but just want to make sure since the previous value is representing a Dash Core version number while the new value is a protocol version number.

Reply:
The 130000 has been here since around the time when @snogcel first forked away from bitcore-p2p and has been introduced here:
https://github.com/dashevo/dashcore-p2p/commit/46c9f918d0391e1db42e9c9b27d4acdc372f40bb

No idea if that was the correct protocol version at that time. But it seems to me that it definitely is not anymore :)

Reply:
Interesting :-) Apparently it didn't cause any problems. This one LGTM.